setup.py

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Installation script for `metaopt.algo.skopt.bayes`."""
+"""Installation script for `orion.algo.skopt.bayes`."""
 from setuptools import setup
 
 setup_args = dict(
-    name='metaopt.algo.skopt.bayes',
+    name='orion.algo.skopt.bayes',
     version=0.1,
     description="Implement a wrapper for skopt bayesian optimizer, for demo and testing.",
     license='BSD-3-Clause',
     author='Xavier Bouthillier',
     author_email='xavier.bouthillier@umontreal.ca',
-    url='https://github.com/bouthilx/metaopt-skopt-bayes',
-    packages=['metaopt.algo.skopt'],
+    url='https://github.com/bouthilx/orion-skopt-bayes',
```

Comment:
url here is wrong

Reply:
Well it was on me, I would have Python package name, to be the same as PyPI package name, same as repo name.

Reply:
You mean orion.algo.skopt and orion-algo-skopt?

Reply:
Let's favor coherence, I agree